src/main/kotlin/no/nav/familie/ks/sak/kjerne/avstemming/KonsistensavstemmingScheduler.kt

```diff
@@ -0,0 +1,48 @@
+package no.nav.familie.ks.sak.kjerne.avstemming
+
+import no.nav.familie.ks.sak.common.EnvService
+import no.nav.familie.ks.sak.kjerne.avstemming.domene.KonsistensavstemmingTaskDto
+import no.nav.familie.leader.LeaderClient
+import no.nav.familie.log.mdc.MDCConstants
+import no.nav.familie.prosessering.internal.TaskService
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.slf4j.MDC
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
+@Service
+class KonsistensavstemmingScheduler(
+    private val envService: EnvService,
+    private val konsistensavstemmingKjøreplanService: KonsistensavstemmingKjøreplanService,
+    private val taskService: TaskService
+) {
+
+    @Scheduled(cron = "\${CRON_KONSISTENS_AVSTEMMING}")
+    @Transactional
+    fun utfør() {
+        MDC.put(MDCConstants.MDC_CALL_ID, UUID.randomUUID().toString())
```

Comment:
Foretrekker å legge til MDC.clear i finally for å rydde opp. Tror Scheduled kjører i en thread pool, så hvis en tråd gjenbrukes så kan det hende at MDC gjenbrukes.